### PR TITLE
Feature aur#142 cleanup sis repository

### PR DESCRIPTION
**Pull request by Xarpagon**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -36,6 +36,8 @@ RUN apt-get update -q && apt-get install -q -y --no-install-recommends \
     python3-stringtemplate3 \
     spin \
     wget \
+    libncurses5 \
```
Comment (Lurkerpas) [inline Dockerfile]: Please note that these are in a more or less alphabetical order, for easier maintenance

File: Dockerfile [hunk 2]
```diff
@@ -67,17 +69,25 @@ RUN apt-get update -q && apt-get install -q -y --no-install-recommends \
...
+    && git checkout 4.2.5.1f \
+    && dotnet build "asn1scc.sln"
 
 # Install opengeode
 RUN git clone https://gitrepos.estec.esa.int/taste/opengeode.git \
     && cd opengeode \
     && PATH=~/.local/bin:"${PATH}" && pyside6-rcc opengeode.qrc -o opengeode/icons.py &&  python3 -m pip install --upgrade .
 
+# Download RTEMS and compile example C application
+RUN wget -q https://rtems-qual.io.esa.int/public_release/rtems-6-sparc-gr712rc-smp-4.tar.xz \
```
Comment (Lurkerpas) [inline Dockerfile:82]: I would split the download and test. Please note that a few lines later there is "Execute tests to see if the image is valid"

File: Dockerfile [hunk 3]
```diff
@@ -67,17 +69,25 @@ RUN apt-get update -q && apt-get install -q -y --no-install-recommends \
...
+    && tar -xf rtems-6-sparc-gr712rc-smp-4.tar.xz -C /opt \
+    && rm -f rtems-6-sparc-gr712rc-smp-4.tar.xz \
+    && cd /opt/rtems-6-sparc-gr712rc-smp-4/src/example \
+    && make
```
Comment (Lurkerpas) [inline Dockerfile]: I think it would be nice if we could add RTEMS bin folder to PATH

Comment (Xarpagon) [reply Dockerfile]: ack

File: Dockerfile (hunk 2 above)
Comment (Xarpagon) [reply Dockerfile:82]: ack

File: Dockerfile (hunk 1 above)
Comment (Xarpagon) [reply Dockerfile]: ack

**Review by Lurkerpas — APPROVED** [review]

**Pull request merged**